projects/product-configuration/src/storefront/variant-configurator/interactive-configuration.module.ts

```diff
@@ -0,0 +1,144 @@
+import { CommonModule } from '@angular/common';
```

Comment:
Let's drop the `storefront` from the path of these files: `projects/product-configuration/src/storefront/variant-configurator/interactive-configuration.module.ts` and `projects/product-configuration/src/storefront/variant-configurator/overview.module.ts`

Reply:
We can discuss the folder structure, but `storefront` should be dropped in this.